### ISSUE TITLE
Api key popup not showing in intellij 2022.2

### ISSUE THREAD

**Issue by Jacob3075** [wakatime/jetbrains-wakatime#228]
I had installed from the intellij setting/plugin sync feature and the api key promt did not show.
I used the cli (~/.wakatime/wakatime-cli) to add the key but that also did not work.
I have tried reinstalling it multiple times.

The last time i reinstalled it I had also removed the .wakatime folder and all the other wakatime files in the home folder, but this time the cli was not installed properly. Its pointing to another file but that file is not present

```
❯ pwd
/home/jacob/.wakatime
❯ lla
lrwxrwxrwx 46 jacob  7 Aug 11:54  wakatime-cli -> /home/jacob/.wakatime/wakatime-cli-linux-amd64

```

Intellij build: `IntelliJ IDEA 2022.2 (Ultimate Edition) Build #IU-222.3345.118, built on July 26, 2022`
OS: `Arcolinux`

Also i have an intel cpu but the cli is pointing to an amd file, idk if that is relevant.
Edit: added `pwd`

**Comment by Jacob3075** [comment]
The api key prompt does not show up for any of the 14.0+ versions. I tried installing version 13.1.14 and the prompt came but the cli file is still pointing to the missing file so the dashboard is not getting updated.
There is no wakatime log file in the home dir or in the .wakatime dir

**Comment by Jacob3075** [comment]
I found issue #227 for the api key and fixed it for 14.0.6 but the cli file is still not working.
Fixed by manually downloading the file and placing it in the correct place.

**Comment by alanhamlett** [comment]
Fixed with db63357ef1ab352c1187f0bf175ac7c968384236 and released in v14.1.0 just now.
